packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -604,8 +609,38 @@ class RawKeyboard {
     _listeners.remove(listener);
   }
 
-  /// A handler for hardware keyboard events that will stop propagation if the
-  /// handler returns true.
+  final List<RawKeyEventHandler> _keyHandlers = <RawKeyEventHandler>[];
+
+  /// Register a handler that is called when the user presses or releases
+  /// a hardware keyboard key if the [keyEventHandler] didn't handle it.
+  ///
+  /// These events will be delivered after the [keyEventHandler] (typically set
+  /// by the [FocusManager]) has returned false. If the [keyEventHandler]
+  /// returns true, then handlers registered in this way will not be called, as
+  /// the key is considered handled already.
+  ///
+  /// If any registered handler returns true, then the key event will be
+  /// considered handled by the platform, but all registered handlers will still
```

Comment:
So when `keyEventHandler` says things are handled, these are not called, but when one of these says things are handled, the other ones are still called (and they will not know that the event was already handled?)? Why the shift in semantics?